app/src/main/java/com/pajato/argus/ListAdapter.kt

```diff
@@ -14,20 +15,38 @@ class ListAdapter(val items: MutableList<Video>) : RecyclerView.Adapter<ListAdap
     }
 
     override fun onBindViewHolder(holder: ViewHolder, position: Int) {
-        val titleTextView: TextView = holder.layout.findViewWithTag("title")
+        // Fill in the data and set an onClick for the delete button
+        val titleTextView = holder.layout.findViewWithTag<TextView>("title")
         titleTextView.text = items[position].title
-        val networkTextView: TextView = holder.layout.findViewWithTag("network")
+        val networkTextView = holder.layout.findViewWithTag<TextView>("network")
         networkTextView.text = items[position].network
+        val deleteButton = holder.layout.findViewWithTag<AppCompatImageButton>("delete")
```

Comment:
I have a hunch this delete button is going to be one action of a few in an overflow (three vertical dots) menu in a card view.

Reply:
We should talk about this in person, but I'm against the idea of putting an overflow menu in the card view when we could have as many as 10 on the screen at a time. It makes things messy and if we can think up a smart way to avoid doing it, I'd prefer that.